app/web/features/search/SearchPage.tsx

```diff
@@ -15,10 +15,17 @@ import {
 import { service } from "service";
 import { GeocodeResult } from "utils/hooks";
 
+import { HostingStatus } from "proto/api_pb";
 import FilterDialog from "./FilterDialog";
 import MapWrapper from "./MapWrapper";
 import SearchResultsList from "./SearchResultsList";
 
+export type typeHostingStatusOptions = Exclude<
+  HostingStatus,
+  | HostingStatus.HOSTING_STATUS_UNKNOWN
+  | HostingStatus.HOSTING_STATUS_UNSPECIFIED
+>[];
+
```

Comment:
Types should generally be all caps so `HostingStatusOptionsType`.

Reply:
done ✅